docs/pages/api-docs/autocomplete.md

```diff
@@ -46,7 +46,7 @@ The `MuiAutocomplete` name can be used for providing [default props](/customizat
 | <span class="prop-name">disabled</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, the input will be disabled. |
 | <span class="prop-name">disabledItemsFocusable</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, will allow focus on disabled items. |
 | <span class="prop-name">disableListWrap</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, the list box in the popup will not wrap focus. |
-| <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | Disable the portal behavior. The children stay within it's parent DOM hierarchy. |
+| <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | Disable the portal behavior. The children stay within its parent DOM hierarchy. |
```

Comment:
"it" refers to "children"

Reply:
This doesn't make much sense because we don't know what the parent is. It makes more sense that "it" refers to the root element rendered by the AutoComplete.

Reply:
```suggestion
| <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | Disable the portal behavior. The DOM hierarchy is left unchanged. |
```

Reply:
This implies that we know what the DOM hierarchy is.

```suggestion
| <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | The `children` will be children of the combobox in the DOM. |
```

> Disable the portal behavior

Is somewhat redundant. To me this is just spelling out the prop name.